addons/web-support/impl/src/main/java/org/jboss/windup/web/addons/websupport/services/WindupExecutorServiceImpl.java

```diff
@@ -87,7 +98,26 @@ public void execute(WindupProgressMonitor progressMonitor, Collection<Path> rule
             configuration.setOptionValue(optionEntry.getKey(), optionEntry.getValue());
         }
 
-        processor.execute(configuration);
+        Logger globalLogger = Logger.getLogger("org.jboss.windup");
+        Handler logHandler = null;
+        try
+        {
+            logHandler = this.logService.createLogHandler(configuration);
+        }
+        catch (IOException e)
+        {
+            throw new WindupException("Failed to create log handler due to: " + e.getMessage(), e);
+        }
+        try
+        {
+            globalLogger.addHandler(logHandler);
```

Comment:
I don't understand why it is isolated into 2 try blocks while the one is more clean and works the same.